types/nes/client.d.ts

```diff
@@ -8,7 +8,7 @@ declare class Client {
     constructor(url: string, options?: Client.ClientOptions);
     onError: (err: any) => void;
     onConnect: () => void;
-    onDisconnect: () => void;
+    onDisconnect: (willReconnect: boolean, log: { code: string, explanation: string, reason: string, wasClean: boolean }) => void;
```

Comment:
[This test](https://github.com/hapijs/nes/blob/master/test/client.js#L133-L155) leads me to believe `code` is `number` instead of `string`, but I haven't actually tested it. Are you positive that it's a string?

Reply:
You are right, thanks for the hint!